Cargo.toml

```diff
@@ -16,12 +16,18 @@ async-trait = "0.1"
 clap = { version = "4.4", features = ["derive", "env"] }
 commit = { git = "https://github.com/EspressoSystems/commit.git" }
 futures = "0.3"
-hotshot = { git = "https://github.com/EspressoSystems/HotShot.git", tag = "0.5.27" }
-hotshot-example-types = { git = "https://github.com/EspressoSystems/HotShot.git", tag = "0.5.27" }
-hotshot-types = { git = "https://github.com/EspressoSystems/HotShot.git", tag = "0.5.27" }
-hotshot-builder-api = { git = "https://github.com/EspressoSystems/HotShot.git", tag = "0.5.27" }
+hotshot = { git = "https://github.com/EspressoSystems/HotShot.git", tag = "0.5.32" }
```

Comment:
Hey, could we sort this?

There's a [cargo tool](https://crates.io/crates/cargo-sort) that will do it, but it has a bug that means you need to manually fix things when you have empty lines before commented lines. The bug was fixed in github 3 months ago, no idea why it hasn't been published since then. Sorting/formatting Cargo.toml files consistently makes merge conflicts less frequent.

Reply:
Thanks for the suggestion. Will be using it frequently in future. https://github.com/EspressoSystems/hs-builder-core/pull/68/commits/832462931d7160e8b11e661e9859098fd5f20cc3